test/e2e/iam/atlas_users_test.go

```diff
@@ -66,10 +66,12 @@ func TestAtlasUsers(t *testing.T) {
 		resp, err := cmd.CombinedOutput()
 		require.NoError(t, err, string(resp))
 
-		var user atlasv2.CloudAppUser
-		require.NoError(t, json.Unmarshal(resp, &user), string(resp))
+		var user *atlasv2.CloudAppUser
```

Comment:
Fix for failing `atlas_iam_e2e` test

Reply:
why making this a pointer fixes it? it should have no meaningful difference

Reply:
That's what I thought but this fixed the e2e. Let me check the values before and after the change. I got some errors so I did not try to fix and see the values.

Reply:
Oh man... a silly issue it is. :D I will just push the correct fix.